src/components/Common/PinRegistrationForm.js

```diff
@@ -86,6 +86,7 @@ const PinRegistrationForm = ({
       })
       return unsubscribe
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
again, `clearPin` being a prop, it should probably be in the deps list (even though it should never change).